docs/validate.md

```diff
@@ -0,0 +1,32 @@
+## Using the Validation Command
+
+The validation command runs a series of checks on a clean Git repository to ensure that the current state of a repository is valid.
+
+### Usage 
+The command accepts either a local (l or local) or a remote (r or remote) flag. These are used to determine whether to validate local or remote charts.
+
+1. **Mode Verification**: The command first establishes whether you're operating in remote or local mode.
+
+2. **Script Options Parsing**: It proceeds to decode your script options file, making sense of all the parameters specified.
+
+3. **Repository Cleanliness Check**: Then, it ensures that your git repository is in a clean state, free from uncommitted changes and ready to process.
+
+4. **Local Flag Operations**: If you have set the local flag:
+    1. **Chart Generation**: It immediately fires up to generate local charts.
```

Comment:
Can you give more details about `generate local charts` or link to some documentation about generating charts ?